src/Servers/IIS/build/Build.Settings

```diff
@@ -4,7 +4,7 @@
      <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">$(MSBuildThisFileDirectory)..\</SolutionDir>
      <Configuration Condition="'$(Configuration)'==''">Debug</Configuration>
      <Platform Condition="'$(Platform)' == ''">Win32</Platform>
-     <PlatformToolset>v141</PlatformToolset>
+     <PlatformToolset>v142</PlatformToolset>
```

Comment:
Ah, I think I missed this one before. When you open the IIS sln now, do you get a warning about needing to upgrade to v142 still?

Reply:
> When you open the IIS sln now, do you get a warning about needing to upgrade to v142 still

You do not.